### PR TITLE
add new meeting specific setting hide metadata background for projector

### LINKED ISSUE #2642: Add new meeting specific setting
Context: https://github.com/OpenSlides/openslides-client/issues/4170 

**Wanted:**
A meeting needs a new setting to make the background from the motion-metadata transparent in the projector.

Recommendation for a name: `motions_hide_metadata_background`
It should be a boolean.

The backend only needs to enable/disable this setting. 

This setting should be optional, there should be no migration.
If the setting is none the client should pretend like the setting is false (I am not sure if that is relevant for the backend-issue).

### PR DESCRIPTION
**Pull request by hjanott**
Closes #2642

### CODE REVIEW

File: global/data/example-data.json [hunk 1]
```diff
@@ -296,6 +296,7 @@
             "motions_enable_reason_on_projector": false,
             "motions_enable_sidebox_on_projector": false,
             "motions_enable_recommendation_on_projector": true,
+            "motions_hide_metadata_background": true,
```
Comment (luisa-beerboom) [inline global/data/example-data.json:299]: Why not use the default?

File: openslides_backend/action/actions/meeting/update.py [hunk 2]
```diff
@@ -102,6 +102,7 @@
     "motions_enable_text_on_projector",
     "motions_enable_reason_on_projector",
     "motions_enable_sidebox_on_projector",
+    "motions_hide_metadata_background",
```
Comment (luisa-beerboom) [inline openslides_backend/action/actions/meeting/update.py:105]: Add the field to one of the tests

**Review by luisa-beerboom — CHANGES_REQUESTED** [review]

File: global/data/example-data.json (hunk 1 above)
Comment (hjanott) [reply global/data/example-data.json]: I changed the value to false.

File: openslides_backend/action/actions/meeting/update.py (hunk 2 above)
Comment (hjanott) [reply openslides_backend/action/actions/meeting/update.py:105]: added

**Review by Elblinator — APPROVED** [review]

**Review by luisa-beerboom — APPROVED** [review]

**Pull request merged**